godtools/App/Features/SetupParallel/SetupParallelLanguageView.swift

```diff
@@ -6,19 +6,19 @@
 //  Copyright © 2021 Cru. All rights reserved.
 //
 
-import Foundation
 import UIKit
 
-class SetupParallelLanguageView: UIViewController {
+class SetupParallelLanguageView: UIViewController, UIGestureRecognizerDelegate {
     
     private let viewModel: SetupParallelLanguageViewModelType
     
-    private var closeButton: UIBarButtonItem?
-
+    private let buttonCornerRadius: CGFloat = 6
+    
+    private var tapRecognizer: UITapGestureRecognizer!
+    
     @IBOutlet weak private var animatedView: AnimatedView!
     @IBOutlet weak private var promptLabel: UILabel!
     @IBOutlet weak private var selectLanguageButtonView: UIView!
-    @IBOutlet weak private var selectLanguageButtonGestureRecognizer: UITapGestureRecognizer!
     @IBOutlet weak private var selectLanguageButtonTitle: UILabel!
     @IBOutlet weak private var yesButton: UIButton!
     @IBOutlet weak private var noButton: UIButton!
```

Comment:
Can you make sure all of these button types are custom type in the .xib.  I think this one is System.  Want to ensure we keep those as custom types otherwise iOS versions could change the appearance of them.